packages/web/src/tools/_typography.scss

```diff
@@ -0,0 +1,40 @@
+@use 'sass:map';
+
+@mixin generate($token) {
+    @if map.has-key($token, font-family) {
+        font-family: unquote(map.get($token, font-family));
+    }
+
+    @if map.has-key($token, font-size) {
+        font-size: map.get($token, font-size);
+    }
+
+    @if map.has-key($token, font-style) {
+        font-style: map.get($token, font-style);
+    }
+
+    @if map.has-key($token, font-weight) {
+        //stylelint-disable-next-line font-weight-notation
+        font-weight: map.get($token, font-weight);
+    }
+
+    @if map.has-key($token, text-transform) {
```

Comment:
I'd write out these optional properties only if they are not `0` or `none`. It's going to bloat our output CSS otherwise:

```css
    letter-spacing: 0;
    text-decoration: none;
    text-indent: 0;
    text-transform: none;
```

Overrides can be done manually after calling this mixin.